server/index.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+
+const bodyParser = require('body-parser');
```

Comment:
I don't think you need body-parser for this code as it's written, but you might need it later. Not sure what your API is going to look like.